Formula/shogun.rb

```diff
@@ -0,0 +1,151 @@
+class Shogun < Formula
+  desc "Large scale machine learning toolbox"
+  homepage "http://www.shogun-toolbox.org/"
+  url "http://shogun-toolbox.org/archives/shogun/releases/6.1/sources/shogun-6.1.3.tar.bz2"
+  sha256 "57169dc8c05b216771c567b2ee2988f14488dd13f7d191ebc9d0703bead4c9e6"
+
+  bottle do
+    sha256 "06aafb46d4cd4e53ebed6f8bb32e1279b5a8b5de9aae2359261cee9de87fc2ca" => :sierra
+    sha256 "e5dd5b447ef40488fb32fa36e2eb943975a6b28db8ae27859e39cb8a87a077a9" => :el_capitan
+    sha256 "b3d320d345da09b55ffb98a59f4fac3b2f6c2672fcbcda693fe0e7a9ee4795a3" => :yosemite
+    sha256 "dfbd03cba5e1a134a520d6c06aceaa3c5143cad638fc208150d980a44e5252cf" => :x86_64_linux
+  end
+
+  patch :DATA
```

Comment:
This should use `patch do` block(s).

Reply:
i've taken this from other formulas, as i want to have the patch part of the formula file

Reply:
Those are legacy.

Reply:
so i'm force to put the patches somewhere separately on the web?

Reply:
Are there upstream commits that apply cleanly? Those can be used like
```
    patch do
      url "https://github.com/bitcoin/bitcoin/commit/1ec0c0a01c.patch?full_index=1"
      sha256 "a1f761fe29f78e783cb4b55f8029900f94b45d1188cb81c80f73347ee2fdc025"
    end
```

Reply:
If not, it needs to go in https://github.com/Homebrew/formula-patches.

Reply:
noup... so i'll have to create one

Reply:
which is another PR? 🤦‍♂️ 
cant i just provide my own patch on my own site?

Reply:
nope. I promise you will live.

Reply:
Your other option is to tag a new release ;)

Reply:
just a constructive feedback: this process really sucks.... you need to fork 2 repos and create 2 different prs for a simple formula.

Reply:
https://github.com/Homebrew/formula-patches/pull/221 is for the patches.... i guess i need to wait until they are merged

Reply:
I did my share of complaining about it at one point. You're not the first to say that and you won't be the last.

Reply:
yep i thought so, i'm just confirming the observation of others ;)

Reply:
Personally I like the fact that it keeps really long patches out of the main repository, which would distort contributor statistics whenever someone has a patch that is thousands of lines long.

Reply:
k i'm ready with the fixes... once the above mentioned pr gets merged i can push the fix for the comments

Reply:
https://raw.githubusercontent.com/Homebrew/formula-patches/9df360c/shogun/fix_python_linking.patch
https://raw.githubusercontent.com/Homebrew/formula-patches/9df360c/shogun/fix_veclib.patch